### PR TITLE
make scope optional and default to a value that won't break

### PR DESCRIPTION
**Pull request by tophercf**
honestly not a big fan of this, but there are cases where db has both empty array and null values in production. addressing, we did see some behavior in dev for a user where the .join statement threw due to undefined value.

### CODE REVIEW

File: src/auth/ConvertFlexbaseToken.ts [hunk 1]
```diff
@@ -29,6 +29,6 @@ export const convertFlexbaseToken = (tokenResponse: FlexbaseTokenResponse): Auth
         tokenType: 'Bearer',
         expiration: future.toUnixInteger(),
         refreshToken: tokenResponse.token,
-        scope: decoded.roles.join(' '),
+        scope: (decoded.roles) ? decoded.roles.join(' ') : '[]',
```
Comment (kyle-b2) [inline src/auth/ConvertFlexbaseToken.ts]: Should the `'[]'` be an empty string instead of array notation in a string?

Comment (tophercf) [reply src/auth/ConvertFlexbaseToken.ts]: it should indeed

Comment (tophercf) [reply src/auth/ConvertFlexbaseToken.ts]: actually, technically i believe if it's a string, it needs to be string empty array, based on what happens with conversions

Comment (tophercf) [reply src/auth/ConvertFlexbaseToken.ts]: nvm, rereviewed - making change

**Review by kyle-b2 — APPROVED** [review]

**Pull request merged**